src/ProfiledAot/README.md

```diff
@@ -0,0 +1,72 @@
+# Profiled AOT support for Android
```

Comment:
This file has the instructions on how to update these files, basically:

    $ ./bin/dotnet/dotnet build -t:Record -r android-arm64 src/ProfiledAot/ProfiledAotMaui/ProfiledAotMaui.csproj
    $ ./bin/dotnet/dotnet build -t:Record -r android-arm64 src/ProfiledAot/ProfiledAotMauiBlazor/ProfiledAotMauiBlazor.csproj